### ISSUE TITLE
Change FAM implementation

### ISSUE THREAD

**Issue by ligi** [ligi/PassAndroid#42]
something on top of design lib - e.g.: https://github.com/konifar

**Comment by guerda** [comment]
Do you mean this lib? https://github.com/konifar/fab-transformation

**Comment by ligi** [comment]
Yes - prepared this in a branch already - but this is a problem: https://github.com/konifar/fab-transformation/issues/7
Quite a lot of device on L now
